### DISCUSSION TITLE
How to extract a pore network model from an image

### DISCUSSION THREAD

**Discussion by HFenice** [PMEAL/porespy#585] [Q&A]
Hello everyone,

I have an image and I want to extract a pore network model from this image and then I want to import it in openpnm. How can I do this?

**Comment by jgostick** [comment]
Hi @HFenice...yes this is absolutely possible.  We have examples on the PoreSpy.org website, but I just looked and realized they are not up-to-date with the latest changes in OpenPNM (I opened an issues and will fix asap).  The best example for you to work from is the unit test in OpenPNM that we added recently:  https://github.com/PMEAL/OpenPNM/blob/dev/tests/integration/PoreSpyIO_on_Berea.py
